web/templates/dropdown_widget_with_label.hbs

```diff
@@ -1,5 +1,5 @@
 <div class="input-group" id="{{widget_name}}_widget_container">
-    <label class="dropdown-title" for="{{widget_name}}_widget">{{label}}
+    <label class="dropdown-title" for="{{widget_name}}-widget">{{label}}
```

Comment:
This commit looks wrong, since these lines that clearly reference the same object are not updated:
```
$ git grep '}_widget'  web/src/
web/src/dropdown_widget.js:        this.widget_id = `#${CSS.escape(widget_name)}_widget`;
web/src/settings_streams.js:        const $stream_dropdown_widget = $(`#${CSS.escape(stream_dropdown_widget_name)}_widget`);
```